### PR TITLE
[2.6] removed rancher logging  0.2.5

### PR DESCRIPTION
**Pull request by paynejacob**

### CODE REVIEW

**Review by nickgerace — DISMISSED** [review]

File: charts/rancher-logging/0.3.0/Chart.yaml [hunk 1]
```diff
@@ -1,12 +0,0 @@
-apiVersion: v1
-description: Rancher logging helm chart to support logging function in rancher
-icon: https://raw.githubusercontent.com/fluent/fluentd-docs/master/public/logo/Fluentd_square.png
-name: rancher-logging
-version: 0.3.0
```
Comment (cbron) [inline charts/rancher-logging/0.3.0/Chart.yaml]: Not deleting 0.3.0, just 0.2.5

**Review by cbron — CHANGES_REQUESTED** [review]

File: charts/rancher-logging/0.3.0/Chart.yaml (hunk 1 above)
Comment (paynejacob) [reply charts/rancher-logging/0.3.0/Chart.yaml]: was 0.3.0 tested?

File: charts/rancher-logging/0.2.5/questions.yml [hunk 2]
```diff
@@ -1 +0,0 @@
-rancher_min_version: 2.5.10-rc1
```
Comment (cbron) [inline charts/rancher-logging/0.2.5/questions.yml:1]: Can you update 0.2.4 max version to 2.5.99

File: charts/rancher-logging/0.3.0/questions.yml [hunk 3]
```diff
@@ -1 +0,0 @@
-rancher_min_version: 2.6.0-rc0
```
Comment (cbron) [inline charts/rancher-logging/0.3.0/questions.yml:1]: 2.6.0-alpha1

**Review by cbron — APPROVED** [review]

File: charts/rancher-logging/0.3.0/Chart.yaml (hunk 1 above)
Comment (cbron) [reply charts/rancher-logging/0.3.0/Chart.yaml]: 0.3.0 was just 0.2.5 copied over, so I'm assuming anything QA'd there is the same

**Pull request merged**